### ISSUE TITLE
Bug in API related to tasks?

### ISSUE THREAD

**Issue by tamuhey** [materialsproject/pymatgen#1252]
## Summary

* `GET https://www.materialsproject.org/rest/v2/tasks/` faild with a error
* the error states `NoneType object is not iterable`

## Example code

```python
url = "https://www.materialsproject.org/rest/v2/tasks/mp-1"
r=requests.get(url, {"API_KEY" : api_key})
```
or

```python
with MPRester(api_key) as m:
    m.get_task_data("mp-48","energy")
```

## Error message

```
Traceback (most recent call last):
  File "/var/www/python/matgen_prod/materials_django/rest/rest.py", line 94, in wrapped
    d = func(*args, **kwargs)
  File "/var/www/python/matgen_prod/materials_django/tasks/rest.py", line 47, in get_property
    entries = qe.get_entries(crit, False, supported_task_properties)
  File "/var/www/miniconda3/envs/mpprod3/lib/python3.6/site-packages/matgendb/query_engine.py", line 305, in get_entries
    symbols = ["{} {}".format(func, label) for label in labels]
TypeError: 'NoneType' object is not iterable
```

## Env
`requests.__version__ = 2.18.4`

**Comment by shyuep** [comment]
@dwinston Can you check on this bug and fix it (with a unittest)? Thanks.

**Comment by dwinston** [comment]
@shyuep bug is fixed on MP backend, and #1254 guards against regression. Can close this.
